tests/test_models.py

```diff
@@ -590,6 +593,28 @@ def _test_score(estimator, p_mat, graph):
     assert np.sum(lik) == estimator.score(graph)
 
 
+class TestEdgeSwaps(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.A = er_np(100, 0.5)
+        cls.B = csr_matrix(cls.A)
+        cls.C = nx.from_numpy_array(cls.A)
+        cls.D = nx.from_scipy_sparse_matrix(cls.B)
+
+    def test_numpy_edge_swap(self):
+        Swapper = EdgeSwapper(self.A)
+        swapped_er, _ = Swapper.swap_edges(n_swaps=100)
+        swapped_er_nx = nx.from_numpy_array(swapped_er)
+        assert list(self.C.degree()) == list(swapped_er_nx.degree())
+
+    def test_scipy_edge_swap(self):
+        Swapper = EdgeSwapper(self.B)
+        swapped_csr, _ = Swapper.swap_edges(n_swaps=100)
+        swapped_csr = swapped_csr.toarray()
+        swapped_csr_nx = nx.from_numpy_array(swapped_csr)
+        assert list(self.D.degree()) == list(swapped_csr_nx.degree())
+
```

Comment:
@kareef928 can we just add one test where you run the same code with the same seed twice and verify that the output is the same both times? I would like to do for both scipy and numpy.